src/latex.jl

```diff
@@ -86,13 +86,21 @@ function animate_latex(text, pos::Point, t, object)
     translate(-pos)
 end
 
```

Comment:
If you don't mind a short docstring would be great :smile:

Reply:
done